package.json

```diff
@@ -97,7 +97,8 @@
     "through2": "0.6.2",
     "url": "^0.11.0",
     "urllib": "^2.3.4",
-    "webpack-stream": "^4.0.3"
+    "webpack-stream": "^4.0.3",
+    "synchronized-promise": "0.3.1"
```

Comment:
Please check and make sure this version is compatible with all node versions that lithium-sdk supports.

Reply:
verified the flow in all supported node versions of lithium-sdk, its working as expected